proto/ibc/applications/middleware/fee/v1/fee.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
```

Comment:
all files need this:
```proto
option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
```
then rerun `make proto-all`

Reply:
I left this out just because I figured it depends on the scaffolding. I'll do you as you suggest tho

Reply:
Ah, I see that it was merged in. makes sense :+1:

Reply:
Updated :+1: